components/sysutils/nut/nut-bins.p5m

```diff
@@ -0,0 +1,111 @@
+
+depend fmri=$(COMPONENT_FMRI)-common@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION) type=require
+
+### NUT Command-line binaries and scripts (daemons, clients)
+
+### Note: While nut-scanner can use snmp, neon/xml, ipmi and usb libs, it pulls
+### them truly dynamically (if available) and skips otherwise. Installing all
+### the NUT driver packages should pull those libs by harder ldd requirements.
+### Conversely if you don't want those UPSes (got no drivers) why scan them? :)
+
+file path=usr/bin/$(MACH64)/nut-scanner
+file path=usr/bin/$(MACH64)/upsc
+file path=usr/bin/$(MACH64)/upscmd
+file path=usr/bin/$(MACH64)/upslog
+file path=usr/bin/$(MACH64)/upsrw
+file path=usr/bin/$(MACH64)/upssched-cmd
+
+file path=usr/bin/nut-scanner
+file path=usr/bin/upsc
+file path=usr/bin/upscmd
+file path=usr/bin/upslog
+file path=usr/bin/upsrw
+file path=usr/bin/upssched-cmd
+
+file path=usr/sbin/$(MACH64)/upsd
+file path=usr/sbin/$(MACH64)/upsdrvctl
+file path=usr/sbin/$(MACH64)/upsmon
+file path=usr/sbin/$(MACH64)/upssched
+
+file path=usr/sbin/upsd
+file path=usr/sbin/upsdrvctl
+file path=usr/sbin/upsmon
+file path=usr/sbin/upssched
+
+# Example notification script for UPSMON
+file files/notifyme path=$(NUTDRVDIR)/notifyme
+
+file path=usr/share/man/man5/hosts.conf.5
+file path=usr/share/man/man5/nut.conf.5
+file path=usr/share/man/man5/ups.conf.5
+file path=usr/share/man/man5/upsd.conf.5
+file path=usr/share/man/man5/upsd.users.5
+file path=usr/share/man/man5/upsmon.conf.5
+file path=usr/share/man/man5/upssched.conf.5
+
+file path=usr/share/man/man8/nut-scanner.8
+file path=usr/share/man/man8/upsc.8
+file path=usr/share/man/man8/upscmd.8
+file path=usr/share/man/man8/upsd.8
+file path=usr/share/man/man8/upsdrvctl.8
+file path=usr/share/man/man8/upslog.8
+file path=usr/share/man/man8/upsmon.8
+file path=usr/share/man/man8/upsrw.8
+file path=usr/share/man/man8/upssched.8
+
+# NUT starter examples are by convention across many distros delivered in /etc
+file path=$(NUTETCDIR)/hosts.conf.sample group=$(NUTGROUP) owner=$(NUTUSER) mode=0640
+file path=$(NUTETCDIR)/nut.conf.sample group=$(NUTGROUP) owner=$(NUTUSER) mode=0644
+file path=$(NUTETCDIR)/ups.conf.sample group=$(NUTGROUP) owner=$(NUTUSER) mode=0644
+file path=$(NUTETCDIR)/upsd.conf.sample group=$(NUTGROUP) owner=$(NUTUSER) mode=0640
+file path=$(NUTETCDIR)/upsd.users.sample group=$(NUTGROUP) owner=$(NUTUSER) mode=0640
+file path=$(NUTETCDIR)/upsmon.conf.sample group=$(NUTGROUP) owner=$(NUTUSER) mode=0640
+file path=$(NUTETCDIR)/upssched.conf.sample group=$(NUTGROUP) owner=$(NUTUSER) mode=0644
+
+# Integration with dynamically added devices (USB mostly)
+# TODO pedantic: split this off to depend on some USB/hotplug/udev package(s)?
+file path=$(NUTHOTPLUGDIR)/usb/libhid.usermap
+file path=$(NUTHOTPLUGDIR)/usb/libhidups
+
+# README also suggests running `udevadm trigger --subsystem-match=usb_device`
+#file path=$(NUTUDEVDIR)/rules.d/52-nut-ipmipsu.rules
+file path=$(NUTUDEVDIR)/rules.d/62-nut-usbups.rules
```

Comment:
Do we have or plan any equivalent? Should they come in samples to get developers interested and started in making a similar solution/integration?